introduction.md

```diff
@@ -2,3 +2,8 @@
 
 ##### Favorite Code School path
 Javascript
+
+##### Deadly skills
+* Git
+* Javascript
+* Ruby
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?